src/main/java/seedu/address/logic/commands/FindAttendanceCommand.java

```diff
@@ -0,0 +1,112 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.util.CollectionUtil.requireAllNonNull;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_CLASS_INDEX;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_LESSON_INDEX;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_STUDENT_INDEX;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_WEEK;
+
+import java.util.List;
+
+import seedu.address.commons.core.Messages;
+import seedu.address.commons.core.index.Index;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.attendance.Attendance;
+import seedu.address.model.attendance.AttendanceRecord;
+import seedu.address.model.attendance.Week;
+import seedu.address.model.lesson.Lesson;
+import seedu.address.model.moduleclass.ModuleClass;
+import seedu.address.model.student.Student;
+
+/**
+ * Finds a student's attendance for a specified lesson in a specified lesson on
+ * a specified week in the student manager.
+ */
+public class FindAttendanceCommand extends Command {
+
+    public static final String COMMAND_WORD = "find-attendance";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + "Finds the attendance of a student in a specified "
+            + "lesson on a specified week.\n"
+            + "Note: All indexes must be positive integers.\n"
+            + "Parameters: "
+            + PREFIX_CLASS_INDEX + "CLASS_INDEX"
+            + PREFIX_LESSON_INDEX + "LESSON_INDEX "
+            + PREFIX_STUDENT_INDEX + "STUDENT_INDEX "
+            + PREFIX_WEEK + "WEEK_NUMBER (must be a positive integer) ";
```

Comment:
Would this be better?
```suggestion
            + "lesson on a specified week.\n"
            + "Note: All indexes and numbers must be positive integers.\n"
            + "Parameters: "
            + PREFIX_CLASS_INDEX + "CLASS_INDEX "
            + PREFIX_LESSON_INDEX + "LESSON_INDEX "
            + PREFIX_STUDENT_INDEX + "STUDENT_INDEX "
            + PREFIX_WEEK + "WEEK_NUMBER";
```